src/pages/EditResourcePage.tsx

```diff
@@ -18,29 +22,71 @@ const StyledEditPublication = styled.div`
 `;
 
 interface EditResourcePageParamTypes {
-  identifier: string;
+  resourceIdentifierFromParam: string;
 }
 
 const EditResourcePage: FC = () => {
-  const { identifier } = useParams<EditResourcePageParamTypes>();
-  const [expanded, setExpanded] = useState<string | false>(false);
   const { t } = useTranslation();
+  const { resourceIdentifierFromParam } = useParams<EditResourcePageParamTypes>();
+
+  const [resourceIdentifier, setResourceIdentifier] = useState<string>(resourceIdentifierFromParam);
+  const [expanded, setExpanded] = useState<string | false>(false);
+  const [showForm, setShowForm] = useState<boolean>(!!resourceIdentifier);
+  const [resourceType, setResourceType] = useState<ResourceCreationType>(ResourceCreationType.FILE);
+
+  const onCreateFile = (resourceIdentifierCreatedOnMainFileSelection: string) => {
+    if (resourceIdentifierCreatedOnMainFileSelection) {
+      setResourceIdentifier(resourceIdentifierCreatedOnMainFileSelection);
```

Comment:
vi kommer nok til å gå for redux likevel pga komplesiteten i skjema - så dette blir kanskje ikke like aktuelt